src/shared/sam/localLambdaRunner.ts

```diff
@@ -15,6 +16,7 @@ import * as pathutils from '../../shared/utilities/pathUtils'
 import { getLogger } from '../logger'
 import { SettingsConfiguration } from '../settingsConfiguration'
 import { recordLambdaInvokeLocal, Result, Runtime, recordSamAttachDebugger } from '../telemetry/telemetry'
+import * as telemetry from '../telemetry/telemetry'
```

Comment:
Double `../telemetry/telemetry` import

Reply:
having stuff like unqualified `Result` is confusing. So I prefixed everything with `telemetry.*`